manage-data/lifecycle/index-lifecycle-management/policy-apply.md

```diff
@@ -0,0 +1,79 @@
+---
+navigation_title: Apply a policy to an index
+applies_to:
+  stack: ga
+products:
+  - id: elasticsearch
+---
+
+# Apply a lifecycle policy to an index [apply-policy-manually]
+
+When you create new {{es}} index you can use an index template to apply the lifecycle policy by which the index will be managed. This process is described in [Configure a lifecycle policy](/manage-data/lifecycle/index-lifecycle-management/configure-lifecycle-policy.md).
+
+You can also manually apply a lifecycle policy to an existing index, as described here. You can do this in {{kib}} or using the {{es}} API.
+
+::::{important}
+Do not manually apply a policy that uses the rollover action. Policies that use rollover must be applied by the index template. Otherwise, the policy is not carried forward when the rollover action creates a new index.
```

Comment:
This statement is good advice & also we apply policies with rollover all the time in Support outside the template. I believe your call out is to not apply a policy to an index where rollover has yet to occur.

Ex: index 1+2 associated to policy A, policy A removed from 1 for manual intervention (ex rehydrating frozen tier), policy A re-added to 1, 1 ILM Move Step pushed past rollover.

Reply:
Thanks for the explanation! I'm still not sure exactly what the warning should be (it's from the current docs), so based on what you've said I've changed it to:

> WARNING: Do not manually apply a policy that uses the rollover action to an index which has not yet rolled over. Otherwise, the policy may not be carried forward when the rollover action creates a new index.